### DISCUSSION TITLE
multiple outlines

### DISCUSSION THREAD

**Discussion by majdgh92** [typst/typst#2380] [Q&A] [answered]
Hello,
I have an outline for content and other outlines for figures and tables.
i couldn't figure out how to show the other outlines(figures and tables) in the main outline(content)
Thanks.

**Comment by EpicEricEE** [comment, marked as answer]
You can use a show-set rule:
```typ
#show outline: set heading(outlined: true)

#outline()
#outline(target: figure.where(kind: image), title: "List of Figures")
#outline(target: figure.where(kind: table), title: "List of Tables")
```

If the main outline should not be outlined, just move the show-set rule somewhere between the main outline and the other ones.

**Reply by majdgh92** [reply]
Thank you very much for reply.
i have figured out away:

#outline(
  title: [#heading(outlined: true)[Abbildungsverzeichnis]],
  indent: auto,
  target: figure.where(kind: image),)

but i would use your way so i don't have to add this heading in each title.
